pkg/services/object/server.go

```diff
@@ -152,6 +153,13 @@ func eACLErr(info aclsvc.RequestInfo, err error) error {
 	return errAccessDenied
 }
 
+const (
+	maxRespMsgSize       = 4 << 20                            // default gRPC limit
```

Comment:
the commit says "inline" but now the object service knows much more about things that were theoretically configurable before. not "inline" imo